docs/metrics.md

```diff
@@ -0,0 +1,92 @@
+# Collecting and Viewing Event Gateway Metrics
+
+This guide details how to collect and analyze metrics from the Event Gateway.
+
+## Contents
+
+1. [Introduction](#introduction)
+1. [Collecting Metrics](#collecting-metrics)
+1. [Visualizing Data](#visualizing-data)
+   1. [Prometheus]#prometheus)
+   1. [InfluxDB](#influxdb)
+1. [List of Metrics](#list-of-metrics)
+   1. [Events API](#events-api)
+   1. [Configuration API](#configuration-api)
+
+### Introduction
```

Comment:
I would move the intro paragraph above the `Contents` and remove the "Introduction" point from TOC.